docs/stencil-docs/customizing-checkout/open-checkout-quick-start.mdx

```diff
@@ -1,6 +1,6 @@
-# Open Checkout Quick Start
+# Open Checkout Tutorial
 
-In this quick start tutorial, we'll configure our development environment and make a code change to a fork of BigCommerce's [Open Checkout](https://github.com/bigcommerce/checkout-js) (also known as [Optimized One-Page Checkout](https://support.bigcommerce.com/s/article/Optimized-Single-Page-Checkout)).
+In tutorial, we'll configure our development environment and make a code change to a fork of BigCommerce's [Open Checkout](https://github.com/bigcommerce/checkout-js) (also known as [Optimized One-Page Checkout](https://support.bigcommerce.com/s/article/Optimized-Single-Page-Checkout)).
```

Comment:
```suggestion
In this tutorial, we'll configure our development environment and make a code change to a fork of BigCommerce's [Open Checkout](https://github.com/bigcommerce/checkout-js) (also known as [Optimized One-Page Checkout](https://support.bigcommerce.com/s/article/Optimized-Single-Page-Checkout)).
```

Reply:
Thanks for the feedback @bc-tgomez. I went ahead and added the missing "the"